reproschema/models/tests/test_item2.py

```diff
@@ -0,0 +1,85 @@
+import os, json, pytest
+from reproschema.models.item import Item
+from reproschema.validate import validate
+
+my_path = os.path.dirname(os.path.abspath(__file__))
+
+item_dir = os.path.join(my_path, "items")
+if not os.path.exists(item_dir):
+    os.makedirs(os.path.join(item_dir))
+
+"""
+Only for the few cases when we want to check against some of the files in
+reproschema/tests/data
+"""
+reproschema_test_data = os.path.join(my_path, "..", "..", "tests", "data")
+
+
+"""
+text items
+"""
+
+
+@pytest.mark.parametrize(
+    "inputType, description, prefLabel, question, responseOptions",
+    [
+        (
+            "text",
+            "text",
+            {"en": "text"},
+            {"en": "question for text item"},
+            {"maxLength": 100, "valueType": "xsd:string"},
```

Comment:
We definitely do not want the user to have to think about `valueType` when generating items, this should be taken care of by the code given that an `inpoutType` pretty much determines the expected  `valueType`